### PR TITLE
Create 82208.md

### PR DESCRIPTION
**Pull request by liu-sz**
Review of 82208

# Summary of Changes

- Please indicate the classes you are modifying: 82208

# Checklist:

Make sure that your contributions:

- [ ] Are accurate and will not mislead the reader
- [ ] Do not encourage or cause academic integrity violations
- [ ] Do not offend professors or unfairly criticize the university

### CODE REVIEW

File: electives/82208.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# 82-208/79-456 Topics in European Studies: Eastern Europe: Society and Culture
+
+| Category |Difficulty  |
+| :------: | :--------: |
+| Reading Assignments | N/A |
```
Comment (mikinty) [inline electives/82208.md]: Why is N/A a difficulty? That doesn't seem very helpful

File: electives/82208.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+| Essays | N/A | 
+| Exams | None | 
+| Prerequisites | N/A | 
+  
+
+This course studies the complex societies and vibrant cultures of Eastern Europe from the Mongol invasion of Europe in the thirteenth century to the end of the Cold War in the late twentieth century. Some topics discussed in this course include nationalism, imperialism, Fascism, globalism, and separatism. The course aims to provide students with a better understanding of the historical processes that led to today’s socio-economic and political landscape of Eastern Europe. No prior knowledge of Slavic languages or Eastern European history is required.
```
Comment (mikinty) [inline electives/82208.md]:
```suggestion
This course studies the complex societies and vibrant cultures of Eastern Europe, from the Mongol invasion of Europe in the thirteenth century, to the end of the Cold War in the late twentieth century. Some topics discussed in this course include nationalism, imperialism, Fascism, globalism, and separatism. The course aims to provide students with a better understanding of the historical processes that led to today’s socio-economic and political landscape of Eastern Europe. No prior knowledge of Slavic languages or Eastern European history is required.
```

File: electives/82208.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Lectures
+
+During lectures, students view documentaries and movies in addition to traditional lectures delivered using slide decks. In Spring 2022, Professor Kats also scheduled tours at the Carnegie Museum of Art and the Nationality Rooms in the Cathedral of Learning for students to gain a first-hand appreciation of Eastern European art and culture. Sometimes there is discussion about the reading assignments, but it is not a main emphasis of the lectures.
+
+  
+
+## Readings
+
+In Spring 2022, students need to read four books: The Bridge On the Drina by Ivo Andric, Journals by Mihail Sebastian, The Book of Laughter and Forgetting by Milan Kundera and Drive Your Plow Over the Bones of the Dead by Olga Tokarczuk. The books are well-selected and represent different historical periods and geographical regions of Eastern Europe. Two of the writers, Ivo Andric from Yugoslavia and Olga Tokarczuk from Poland are Nobel Prize in Literature laureates. Reading these books is enjoyable and should not be very time consuming (~6 hours per week).
```
Comment (mikinty) [inline electives/82208.md]: Can you format the 4 books to be more easily readable? An enumerated list would be good, like 

1. Book one
2. Book two
3. Book three

File: electives/82208.md (hunk 1 above)
Comment (mikinty) [reply electives/82208.md]: Eg some things to consider are about whether the essays are graded hard or the hw. Are the readings hard? Etc

File: electives/82208.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+  
+
+## Essays
+
+Students need to write four 3-page book reports on the books they read, with each accounting for 20% of their final grade. Class participation accounts for 20%, too. Students need to assess the arguments, logic, and design of the books, but a formal literary analysis is not required.
```
Comment (mikinty) [inline electives/82208.md]: How do you succeed with these essays? Are they easy? Can you do it in one night? Are there OH or some other way of getting support?

**Review by mikinty — CHANGES_REQUESTED** [review]
Would like some more detail about the class, and also more detailed judgment about difficulty

**Comment by mikinty** [comment]
Also please add a link to the main README table of contents

**Comment by liu-sz** [comment]
> Also please add a link to the main README table of contents

Added

**Review by mikinty — APPROVED** [review]
Thanks!

**Pull request merged**